test/distributed/org/apache/cassandra/distributed/test/HintsMaxSizeTest.java

```diff
@@ -99,6 +172,36 @@ public void testMaxHintedHandoffSize() throws Exception
         }
     }
 
+    private void waitUntilNoHints(IInvokableInstance node1, UUID node2UUID)
+    {
+        await().pollInterval(10, SECONDS)
+               .timeout(5, MINUTES)
```

Comment:
The timeout should probably be lower so if it times out we don't have to wait 5 minutes to find out